packages/web-component-tester/gulpfile.js

```diff
@@ -72,112 +73,114 @@ gulp.task('clean', (done) => {
   done();
 });
 
-gulp.task('test', function (done) {
-  runSequence(
-    'build:typescript-server',
-    'lint',
-    'test:unit',
-    'test:integration',
-    done);
+const tsProject = ts.createProject('tsconfig.json', {typescript});
+gulp.task('build:typescript-server', function() {
+  // Ignore typescript errors, because gulp-typescript, like most things
+  // gulp, can't be trusted.
+  return tsProject.src()
+      .pipe(tsProject(ts.reporter.nullReporter()))
+      .js.pipe(gulp.dest('./'));
 });
 
-gulp.task('build-all', (done) => {
-  runSequence('clean', 'lint', 'build', done);
+const browserTsProject =
+    ts.createProject('browser/tsconfig.json', {typescript});
+gulp.task('build:typescript-browser', function() {
+  return browserTsProject.src()
+      .pipe(browserTsProject(ts.reporter.nullReporter()))
+      .js.pipe(gulp.dest('./browser/'));
 });
 
-gulp.task('build',
-  ['build:typescript-server', 'build:browser', 'build:wct-browser-legacy']);
-
-const tsProject = ts.createProject('tsconfig.json', { typescript });
-gulp.task('build:typescript-server', function () {
-  // Ignore typescript errors, because gulp-typescript, like most things
-  // gulp, can't be trusted.
-  return tsProject.src().pipe(tsProject(ts.reporter.nullReporter())).js.pipe(gulp.dest('./'));
+gulp.task('test', function(done) {
+  runSequence(
+      'build:typescript-server', 'lint', 'test:unit', 'test:integration', done);
 });
 
-const browserTsProject = ts.createProject('browser/tsconfig.json', {
-  typescript
-});
-gulp.task('build:typescript-browser', function () {
-  return browserTsProject.src().pipe(
-    browserTsProject(ts.reporter.nullReporter())).js.pipe(gulp.dest('./browser/'));
+gulp.task('build-all', (done) => {
+  runSequence('clean', 'lint', 'build', done);
 });
 
 // Specific tasks
 
-gulp.task('build:browser', ['build:typescript-browser'], function (done) {
-  rollup.rollup({
-    entry: 'browser/index.js',
-  }).then(function (bundle) {
-    bundle.write({
-      indent: false,
-      format: 'iife',
-      banner: fs.readFileSync('browser-js-header.txt', 'utf-8'),
-      intro: 'window.__wctUseNpm = false;',
-      dest: 'browser.js',
-      sourceMap: true,
-      sourceMapFile: path.resolve('browser.js.map')
-    }).then(function () {
-      done();
+gulp.task(
+    'build:browser', gulp.series(['build:typescript-browser']), function(done) {
+      rollup
+          .rollup({
+            entry: 'browser/index.js',
+          })
+          .then(function(bundle) {
+            bundle
```

Comment:
Pre-existing issue, but we're missing a return here.

I think we can use async/await in this gulpfile, as it was added in Node 7 and Node 8 is the latest LTS release. (Maybe not worth switching to async/await in this change, but a good longer term goal)

Reply:
The missing return is needed so that errors are caught by the outer .catch